### PR TITLE
moved the AuthOptions to the configurator appsettings.json and use it through DI where it is used  *moved the connection path to DB in appsettings.json

### PR DESCRIPTION
**Pull request by BadikVV**
added authentication and DB configurations

### CODE REVIEW

File: MakeGreyImageAPI/Services/AuthService.cs [hunk 1]
```diff
@@ -13,13 +13,17 @@ public class AuthService
 {
```
Comment (lordspinach) [inline MakeGreyImageAPI/Services/AuthService.cs]: Лишняя пустая строка

File: MakeGreyImageAPI/Program.cs [hunk 2]
```diff
@@ -18,13 +18,17 @@
 using ILogger = MakeGreyImageAPI.Interfaces.ILogger;
 
 var builder = WebApplication.CreateBuilder(args);
+var configuration = new ConfigurationManager().AddJsonFile("appsettings.json").Build();
+var authOptions = configuration.GetSection("AuthOptions").Get<AuthOptions>();
+
```
Comment (lordspinach) [inline MakeGreyImageAPI/Program.cs:23]: Лишняя пустая строка

**Review by lordspinach — CHANGES_REQUESTED** [review]

File: MakeGreyImageAPI/Services/AuthService.cs (hunk 1 above)
Comment (BadikVV) [reply MakeGreyImageAPI/Services/AuthService.cs]: Удалил

File: MakeGreyImageAPI/Program.cs (hunk 2 above)
Comment (BadikVV) [reply MakeGreyImageAPI/Program.cs:23]: Удалил

**Review by lordspinach — APPROVED** [review]

**Pull request merged**